task_templates/README.md

```diff
@@ -1,50 +1,73 @@
 # Templates for Custom Tasks
+
 In this directory are three folders - estimators, transforms, and pipelines
 
 * estimators - meant to be used as the prediction step in a blueprint
-* transforms - meant to be used as a preprocessing step
-* pipelines - meant to be added as a single task blueprint, these have some preprocessing and an estimator built in
+* transforms - meant to be used as a preprocessing step in a blueprint
+* pipelines - meant to be added as a single task blueprint, these have some preprocessing and an
```

Comment:
nit:
these have some preprocessing and an estimator built in -> these have preprocessing and an estimator built into a single task